CODEOWNERS

```diff
@@ -2,6 +2,7 @@ server/chat-plugins/othermetas.ts @KrisXV @TheImmortal
 server/chat-plugins/random-battles.ts @KrisXV @TheImmortal
 server/chat-plugins/scavenger*.ts @xfix @sparkychildcharlie
 server/chat-plugins/trivia.ts @AnnikaCodes
+server/chat-plugins/help.ts @mia-pi-git
 server/chat-plugins/hosts.ts @AnnikaCodes
```

Comment:
CODEOWNERS should probably be alphabetized by file name?

Reply:
Alternatively, we could alphabetize by GitHub username, but this does neither.

Reply:
Filename makes more sense to me.